modules/ob-release-notes-1-0.adoc

```diff
@@ -0,0 +1,46 @@
+// This module is included in the following assemblies:
+// * builds/ob-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="ob-release-notes-1-0_{context}"]
+= Release notes for {builds-title} General Availability 1.0
+
+{builds-title} General Availability (GA) 1.0 is now available on {ocp-product-title} 4.14.
+
+[id="new-features-1-0_{context}"]
+== New features
+
+The following sections highlight what is new in {builds-title} 1.0.
+
+[id="builds-new-features-1-0_{context}"]
+=== Builds
+
+* This release supports the `buildah` and `source-to-image` build strategies.
++
+[NOTE]
+====
+The `buildpacks` build strategy is currently in Developer Preview. It is planned to make this strategy generally available for use in a future release.
+====
+
+* With this release, you can define volume mounts in your `BuildStartegy` resource. When defining a strategy, you can declare volumes which can be shared across build steps and build runs. You can also declare the mount point for the volume which is fixed across all build steps. Volumes can either have a fixed volume source, or an overridable volume source that you can set in a `Build` or `BuildRun` resource.
+
+* With this release, you can embed a complete build specification into your `BuildRun` resource by using the `spec.build.spec` field. This specification embedding allows you to build an image without creating and maintaining a dedicated `Build` resource.
+
+* This release supports automated cleaning of completed build runs after a specific time or when a certain number of build runs is reached. By using retention parameters, you can specify the duration for which a completed build run can exist and the number of succeeded or failed build runs that can exist.
+
+* With this release, you can configure a build by defining the source, build strategy, parameter values, builder or docker file, output, retention parameters, and volumes in a `Build` resource.
+
+* With this release, you can configure a build strategy by defining strategy parameters, system parameters, step resources definitions, annotations, and volumes in a `BuildStrategy` or `ClusterBuildStrategy` resource.
+
+* With this release, you can configure a build run by defining the build reference, build specification, parameter values, service account, output, retention parameters, and volumes.
```

Comment:
Should each release note link to official feature docs or an engineering Jira?

Reply:
I would definitely consider this when certain things are finalized in this product context.